### PR TITLE
build for OpenBLAS 0.3.15

### PR DESCRIPTION
**Pull request by mattip**

### CODE REVIEW

**Comment by tylerjereddy** [comment]
I wonder if we'll be able to bump SciPy up soon--I got stuck with a Windows problem on the last few releases of OpenBLAS: https://github.com/scipy/scipy/pull/13270

**Comment by mattip** [comment]
@matthew-brett any idea what is going on with the windows 64 bit build? The error is 
```
++ find . -maxdepth 1 -type f -name '*.a' '!' -name '*.dll.a' -printf '%P\n'
+ static_libname='libopenblas64__v0.3.15-gcc_8_1_0.a
libopenblas64__v0.3.15-gcc_8_1_0p-r0.3.15.a'
+ set -x
+ make -C exports INTERFACE64=1 SYMBOLSUFFIX=64_ objcopy.def
mingw32-make: Entering directory 'D:/a/openblas-libs/openblas-libs/OpenBLAS/exports'
perl ./gensymbol objcopy x86_64 _ 0 0  0 0 0 0 "" "64_" 1 0 1 1 1 1 > objcopy.def
mingw32-make: Leaving directory 'D:/a/openblas-libs/openblas-libs/OpenBLAS/exports'
+ objcopy --redefine-syms exports/objcopy.def 'libopenblas64__v0.3.15-gcc_8_1_0.a
libopenblas64__v0.3.15-gcc_8_1_0p-r0.3.15.a' 'libopenblas64__v0.3.15-gcc_8_1_0.a
libopenblas64__v0.3.15-gcc_8_1_0p-r0.3.15.a.renamed'
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\objcopy.exe: 'libopenblas64__v0.3.15-gcc_8_1_0.a
libopenblas64__v0.3.15-gcc_8_1_0p-r0.3.15.a': No such file
```

**Comment by matthew-brett** [comment]
Sorry - I don't know much about MinGW.  Am I reading it right, that the
'find' is finding two ".a" files, and that this is leading to the error?

**Comment by isuruf** [comment]
>  Am I reading it right, that the 'find' is finding two ".a" files, and that this is leading to the error?

Yes, that is correct.

**Comment by mattip** [comment]
So the question becomes why are we getting both `libopenblas64__v0.3.15-gcc_8_1_0p-r0.3.15.a` and `libopenblas64__v0.3.15-gcc_8_1_0.a`.

**Comment by isuruf** [comment]
`tools/build_gfortran.sh` should be fixed too.

File: tools/build_openblas.sh [hunk 1]
```diff
@@ -78,8 +78,8 @@ DLL_BASENAME=libopenblas${SYMBOLSUFFIX}_${LIBNAMESUFFIX}
 if [ "$INTERFACE64" == "1" ]; then
     # OpenBLAS does not build a symbol-suffixed static library on Windows:
     # do it ourselves
-    static_libname=`find . -maxdepth 1 -type f -name '*.a' \! -name '*.dll.a' -printf '%P\n'`
     set -x  # echo commands
+    static_libname=$(find . -maxdepth 1 -type f -name '*.a' \! -name '*.dll.a' | head 0)
```
Comment (isuruf) [inline tools/build_openblas.sh:82]:
```suggestion
    static_libname=$(find . -maxdepth 1 -type f -name '*.a' \! -name '*.dll.a' | head -1)
```

Comment (mattip) [reply tools/build_openblas.sh]: thanks, done.

**Comment by judahrand** [comment]
Infinite monkeys win again! 🎉🎉🎉

No criticism here, blind change or not! Working is working and hopefully can see 0.3.15 available soon. Should hopefully help unblock `scipy` M1 wheels.

**Pull request merged**